database/src/main/java/com/hover/stax/database/StaxDatabase.kt

```diff
@@ -0,0 +1,5 @@
+package com.hover.stax.database
+
+import androidx.room.RoomDatabase
+
+abstract class StaxDatabase : RoomDatabase()
```

Comment:
This is not implemented anywhere

Or is it a placeholder to use when we migrate `AppDatabase`?

Reply:
Great